### PR TITLE
docs (workflows): Image singularity

### PR DESCRIPTION
**Pull request by Gorkem-Encord**

### CODE REVIEW

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## Image singularity
+
+This metric checks all your images in the dataset and assigns a score to them, which shows
+their uniqueness.
+
+- Uniqueness score is in the range of [0,1]. A higher value means a more unique image.
+- A zero value indicates that this image has at least one identical image in this dataset. In the case of _N_ 
+duplicate images, we assign a value of zero to _N-1_ of them (only one of them will have a nonzero value) so that they
+can be excluded from the dataset immediately.
+- In the case of duplicate images, _N-1_ image will be linked to a single image among those and this image can be
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:24]:
```suggestion
- In the case of duplicate images, _N-1_ images will be linked to a single image among those and this image can be 
```

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+duplicate images, we assign a value of zero to _N-1_ of them (only one of them will have a nonzero value) so that they
+can be excluded from the dataset immediately.
+- In the case of duplicate images, _N-1_ image will be linked to a single image among those and this image can be 
+visualized using the `Description` field under each image.
+- Near duplicate images are labeled as `Near duplicate image` and shown side by side in the grid view. So that
+users can choose which one to keep and which to remove.
+
+## Steps
+Navigate to the _Data Quality_ > _Explorer_  tab and select the **Image Singularity** quality metric in the top left 
+combobox to order your data.
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:31]:
```suggestion
dropdown to order your data. 
```

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+![remove-duplicateimages-1](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-1.png)
+
+In the dashboard, you can see the distribution of your data according to the image singularity scores.
+
+There are approximately 200 duplicate images in this example.
+
+![remove-duplicateimages-2](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-2.png)
+
+Go a little bit down and see the duplicate images. If you click **show similar images** button, you should see at least
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:41]:
```suggestion
Scroll down to see the duplicate images. If you click the **show similar images** button, you should see at least
```

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+one image in the similar ones. If you have [Encord Annotate](https://encord.com/) account, you can directly inspect the other image via
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:42]: I think that this is more a matter of whether the project is associated to Encord Annotate right?

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+the link in the description.
+
+![remove-duplicate-images-3](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-3.png)
+
+Now we want to remove all these duplicate images and imporve the quality of our dataset. To do that, first we will
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:47]:
```suggestion
Now, we want to remove all these duplicate images and improve the quality of our dataset. To do that, first we will 
```
Here we go from "you do this and that" to "we want to ...". We should keep the writing in the "you form".

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- Create a new Encord Active project.
+
+### Removing near duplicate images
+
+_An example of near duplicate image pairs detected with Encord Active_
+![remove-duplicate-images-11](../../images/workflows/improve-your-data-and-labels/remove-duplicate-images/remove-duplicate-images-11.png)
+
+
+Near duplicate images are the ones where one image is a slightly shifted/blurred/distorted version of the other. So, they 
+should also be removed from the dataset; however, this time you now need to choose which one to stay and which to go. They
```
Comment (frederik-encord) [inline docs/docs/workflows/improve-your-data/remove-duplicate-images.md:86]:
```suggestion
should also be removed from the dataset; however, this time you need to decide which sample gets to stay and which has to go. They 
```

**Review by frederik-encord — CHANGES_REQUESTED** [review]

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md (hunk 5 above)
Comment (Gorkem-Encord) [reply docs/docs/workflows/improve-your-data/remove-duplicate-images.md:47]: 👍

File: docs/docs/workflows/improve-your-data/remove-duplicate-images.md (hunk 4 above)
Comment (frederik-encord) [reply docs/docs/workflows/improve-your-data/remove-duplicate-images.md:42]: My point was doing something like this:
```suggestion
one image in the similar ones. If you have a project which is imported from [Encord Annotate](https://encord.com/), you can directly inspect the other image via
```

**Review by frederik-encord — APPROVED** [review]

**Pull request merged**